### ISSUE TITLE
What are the [M] in the prompt? In figure 3. the illustration for the Zer-shot classification and QA task?

### ISSUE THREAD

**Issue by WEIYanbin1999** [zjukg/KGTransformer#1]

**Comment by wencolani** [comment]
It refers to the mask token, the same as the mask token used in subgraph Pre-training.

**Comment by WEIYanbin1999** [comment]
Thanks for you answer. Besides, I try to run the pre-train code: "python run_pretrain.py --pretrain_dataset BIG --dataset_name BIG --num_hidden_layers 4 --train_bs 16 --lr 1e-4 --epochs 10" but the process frozen after "The two_hop_triple_path not exists, generate and dump it":

![image](https://user-images.githubusercontent.com/127707751/231987270-d6fa4f6e-5068-444a-ad92-2b56e71249ec.png)
could you help me?

**Comment by YushanZhu** [comment]
When first run, it needs to generate and save subgraphs file, this process may take a few more moments to complete.
